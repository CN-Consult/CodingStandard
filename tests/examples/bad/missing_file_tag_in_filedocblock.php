<?php
/**
 *
 *
 * @version 2.11
 * @copyright 2020 CN-Consult GmbH
 * @author Daniel Haas <daniel.haas@cn-consult.eu>
 */
 