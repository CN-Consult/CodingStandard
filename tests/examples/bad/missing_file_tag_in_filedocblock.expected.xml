<?xml version="1.0" encoding="UTF-8"?>
<phpcs version="does not matter">
    <file name="examples/bad/missing_file_tag_in_filedocblock.php" errors="1" warnings="0" fixable="0">
        <error line="8" column="2" source="..FileComment.MissingFileTag" severity="5" fixable="0">Missing @file tag in file comment</error>
    </file>
</phpcs>
